src/fsharp/vs/service.fs

```diff
@@ -1482,8 +1483,8 @@ type TypeCheckInfo
             | CNR(_, (Item.CustomBuilder _ | Item.CustomOperation _), ItemOccurence.Use, _, _, _, m) ->
                 Some (m, SemanticClassificationType.ComputationExpression)
             // well known type aliases get colored as keywords
-            | CNR(_, (Item.Types (n, _)), _, _, _, _, m) when keywordTypes.Contains(n) ->
-                Some (m, SemanticClassificationType.IntrinsicType)
```

Comment:
> is there a good reason to color them as keywords? 

They are intrinsic types. I'd like to add `option` and `result` to the list as well. And may be more.